DESCRIPTION

```diff
@@ -1,39 +1,40 @@
-Package: mhealthtools
```

Comment:
Let's keep the package name lowercase. All of the major packages conform to this style and don't use camel case (Everything in tidyverse, everything in r-lib, roxygen2, etc).